facebook_scraper/utils.py

```diff
@@ -43,3 +46,36 @@ def make_html_element(html: str, url=DEFAULT_URL) -> Element:
 
 def html2text(html: str) -> str:
     return _html2text(html)
+
+
+month = r"Jan(?:uary)?|" \
+        r"Feb(?:ruary)?|" \
+        r"Mar(?:ch)?|" \
+        r"Apr(?:il)?|" \
+        r"May|" \
+        r"Jun(?:e)?|" \
+        r"Jul(?:y)?|" \
+        r"Aug(?:ust)?|" \
+        r"Sep(?:tember)?|" \
+        r"Oct(?:ober)?|" \
+        r"Nov(?:ember)?|" \
+        r"Dec(?:ember)?|" \
+        r"Yesterday|" \
+        r"Today"
+date = f"({month}) " + r"\d{1,2}"
+hour = r"\d{1,2}"
+minute = r"\d{2}"
+period = r"AM|PM"
+exact_time = f"({date}) at {hour}:{minute} ({period})"
```

Comment:
Again, I think it shouldn't capture the date or period.
```suggestion
exact_time = f"(?:{date}) at {hour}:{minute} (?:{period})"
```

Reply:
I'm not sure I understand, why again shouldn't the whole date and am\pm be captured?

Reply:
Because those groups are not being used.

In `parse_datetime` only the group 0 is being retrieved, which is whole matched string.
It would make sense to capture every element of the date separately if we were parsing the date ourselves, but we are just passing the whole string to dateparser.

Reply:
I see. Thanks!